include/CppUTestExt/MockCheckedExpectedCall.h

```diff
@@ -36,10 +36,12 @@ class MockCheckedExpectedCall : public MockExpectedCall
 
 public:
     MockCheckedExpectedCall();
+    MockCheckedExpectedCall(unsigned int numCalls);
     virtual ~MockCheckedExpectedCall();
 
     virtual MockExpectedCall& withName(const SimpleString& name) _override;
-    virtual MockExpectedCall& withCallOrder(unsigned int callOrder) _override;
+    virtual MockExpectedCall& withCallOrder(unsigned int callOrder) _override { return withCallOrder(callOrder, callOrder); }
```

Comment:
Maybe @arstrube could give his opinion on this. Shall I fix that in this PR, or is it OK to leave it for a short while, taking in account that the next 2 PRs that I've planned are cleanup ones, where actually [that superfluous method is removed](https://github.com/jgonzalezdr/cpputest/commit/1855c48c08d1a50248d467b8900d9938474ea307)?